docs/user/whats-new.asciidoc

```diff
@@ -17,8 +18,6 @@ releases, refer to:
 For the full list of changes in {minor-version}, see <<release-notes>> and
 <<breaking-changes>>. 
 
-coming[7.9.0]
```

Comment:
As this is the 7.x branch, should this be "coming[7.10.0]"

Reply:
Ah, I didn't see your comment til after the merge, sorry.  If you want to add new coming tags, they'll need to be in a new PR.